src/proto/h2/client.rs

```diff
@@ -105,6 +106,11 @@ where
         .await
         .map_err(crate::Error::new_h2)?;
 
+    debug!(
+        server.max_concurrent_send_streams = conn.max_concurrent_send_streams(),
```

Comment:
So I just wanted to talk through the naming here, cause it twisted me a little. This is in he `client` module, and logs this as the *server's* max send streams... But is this option actually the max streams the *server* can send? I thought `conn.max_concurrent_send_streams()` was how many this connection is allowed to send (so in this module, the client's send limit). Or am I all twisted up?

Reply:
it's the number of streams that the server _told us_ we can send. i wanted to make it clear that the setting was _coming_ from the server, but yeah, after thinking about it, maybe this does not actually suggest the correct meaning. i'll change that.